src/Middleware/Diagnostics.EntityFrameworkCore/src/Views/DatabaseErrorPage.cshtml

```diff
@@ -115,7 +115,7 @@
                     };
 
                     var formBody = "context=" + encodeURIComponent(document.getElementById('applyMigrations').getAttribute('data-assemblyname'));
```

Comment:
Is this intentional?

Reply:
Pardon? I didn't change this. I don't know whether it is right or not 🤷‍♂️

Reply:
Oh nevermind. I misread the syntax highlighting as a diff:

![image](https://user-images.githubusercontent.com/174281/103159024-27d7c880-4779-11eb-94a0-3d58d99794e2.png)